contributor_payments.csv

```diff
@@ -0,0 +1 @@
+Tx hash, Tx date, Contributor, Contributor Address, Work effort (hrs), amount in USD, VWAP, amount in XTZ, Reason for payment, Comment
```

Comment:
This means we add the compensation for the review as a separate line - I am ok with that, I would add "git commit hashs" with ; or , separators.
Put Contributor and Contributor Address first to keep a read only list of contributors and their permanent xtz addresses at the top of the list. you can transfer the comments from the CONTRIBUTORS file to the comment here

Reply:
I made an update with the other changes, I will update `contributor_payments.csv` with the history as well once the first payments are made.